src/types/index.ts

```diff
@@ -1269,7 +1286,7 @@ export type PermissionsLike = {
     }
 );
 
-export interface PortfolioMovement {
+export interface PortfolioMovementFungible {
```

Comment:
```suggestion
export interface FungibleAssetPortfolioMovement {
```
This feels more clear.